asp.tf

```diff
@@ -0,0 +1,18 @@
+locals {
+
+  ase_name = "cmc-${var.env}"
+  common_tags = {
+  }
+}
+
+module "appServicePlan" {
```

Comment:
asp_sku_size should be set, probably I2 to match what it is currently otherwise if we're downgrading to I1 to save money we should run a perf test before deploying it to prod